tests/functional/test_basic.py

```diff
@@ -195,7 +189,9 @@ def test_python_makefile(pyi_builder):
 
 
 def test_set_icon(pyi_builder, data_dir):
-    icon_dir = os.path.join(data_dir, 'icons')
+    # Note that the name test_set_icon (not just test_icon) for prevents
```

Comment:
I would have renamed the folder `tests/functional/data/icons` to `set_icon` to solve this instead. Likewise for `django_site` to `django` or `django/django_site`